### PR TITLE
Adding to roboconf-core a class that parses a Dockerfile

### PR DESCRIPTION
**Pull request by diarraa**
This class allows us to parse a Dockerfile.

### CODE REVIEW

File: core/roboconf-core/src/main/java/net/roboconf/core/DockerfileParser.java [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+			}
+
+		} finally {
+			Utils.closeQuietly(br);
+		}
+		return result;
+	}
+
+	/**
+	 * Executes a docker command on the VM and prints on the console its output.
```
Comment (vincent-zurczak) [inline core/roboconf-core/src/main/java/net/roboconf/core/DockerfileParser.java]: Please, review the formating of these Javadoc comments.

File: core/roboconf-core/src/main/java/net/roboconf/core/DockerfileParser.java [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+	/**
+	 * Copies files using a Docker command passed in parameter as a string.
+	 *
+	 * @param command
+	 *            a string (not null)
+	 * @throws IOException
+	 *             if the file could not be created
+	 */
+	public static void copyFiles(String command) throws IOException {
```
Comment (vincent-zurczak) [inline core/roboconf-core/src/main/java/net/roboconf/core/DockerfileParser.java]: Useless with respect to parsing Dockerfiles. Please, remove it.
We will see how to deal with execution later, as it may vary depending on the context.

File: core/roboconf-core/src/main/java/net/roboconf/core/utils/ProgramUtils.java [hunk 3]
```diff
@@ -92,6 +92,7 @@ public static int executeCommand(
 		return exitValue;
 	}
 
+
```
Comment (vincent-zurczak) [inline core/roboconf-core/src/main/java/net/roboconf/core/utils/ProgramUtils.java]: Why this new line?

File: core/roboconf-core/src/test/java/net/roboconf/core/DockerfileParserTest.java [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+		Assert.assertEquals(DockerfileParser.DockerCommandType.ADD, c4.type);
+		Assert.assertEquals("rabbitmq-env.conf /etc/rabbitmq/rabbitmq-env.conf", c4.argument);
+
+		Assert.assertEquals(DockerfileParser.DockerCommandType.COPY, c5.type);
+		Assert.assertEquals("/ttee/tree/toto.txt /usr/local/deploy/", c5.argument);
+
+	}
+
+	@Test
+	public void executeDockerCommandTestWindows() throws IOException, InterruptedException{
```
Comment (vincent-zurczak) [inline core/roboconf-core/src/test/java/net/roboconf/core/DockerfileParserTest.java]: Please, remove the unit tests of the removed methods.

File: pom.xml [hunk 5]
```diff
@@ -25,7 +25,6 @@
 		<version>1.5</version>
 	</parent>
 
-	<groupId>net.roboconf</groupId>
```
Comment (vincent-zurczak) [inline pom.xml]: Why did you remove the group ID?

**Comment by vincent-zurczak** [comment]
There are still comments that were not taken into account.

File: pom.xml (hunk 5 above)
Comment (diarraa) [inline pom.xml]: It was be deleted by Eclipse during a quick fix. I'll correct it.

File: karaf/roboconf-karaf-dist-agent/pom.xml [hunk 6]
```diff
@@ -133,5 +133,78 @@
 				</configuration>
 			</plugin>
 		</plugins>
+		<pluginManagement>
```
Comment (vincent-zurczak) [inline karaf/roboconf-karaf-dist-agent/pom.xml]: To remove...
